src/configure

```diff
@@ -4456,7 +4460,7 @@ rm -f core conftest.err conftest.$ac_objext \
 $as_echo "found" >&6; }
                         LRT="-lrt -lpthread"
                 else
-			as_fn_error $? "cannot find timer_create and timer_*ettime symbols neither in the base system libraries nor in -lrt, nor in -lrt -lpthread" "$LINENO" 5
+			as_fn_result $? "cannot find timer_create and timer_*ettime symbols neither in the base system libraries nor in -lrt, nor in -lrt -lpthread" "$LINENO" 5
```

Comment:
I'm not sure where "`as_fn_result`" originates from. If I rerun autoconf on configure.in, it disappears and configure works fine. However, without it, I receive the following error:

> checking for timer_create and timer_*ettime symbols in -lrt -lpthread... ./configure: line 4463: as_fn_result: command not found

Since we need to regenerate the configure file anyway, could you please make the following change to line 232 in configure.in before rerunning autoconf?:

```
diff --git a/src/configure.in b/src/configure.in
index 53f3dd63..c04fa823 100644
--- a/src/configure.in
+++ b/src/configure.in
@@ -232,7 +232,7 @@ else
                         AC_MSG_RESULT([found])
                         LRT="-lrt -lpthread"
                 else
-                       AC_MSG_RESULT([cannot find timer_create and timer_*ettime symbols neither in the base system libraries nor in -lrt, nor in -lrt -lpthread])
+                       AC_MSG_RESULT([not found])
                fi
        fi
 fi
```

Reply:
The configure.in line originate from allowing the script to pass even if the condition is not met. This pass is needed for macos to compile papi. This was a solution brought up during the talk about disabling libsde with Anthony.

Regarding the configure line, I manually interpreted the line(I should not have) instead of using what configure.in generated. I ran autoconf to generate what ```
AC_MSG_RESULT([cannot find timer_create and timer_*ettime symbols neither in the base system libraries nor in -lrt, nor in -lrt -lpthread])
+                       AC_MSG_RESULT([not found])
```would have generated and have appended that to the configure script.

Reply:
I have made the change, so it does not print the entire "cannot find timer_create and timer_*ettime symbols neither in the base system libraries nor in -lrt, nor in -lrt -lpthread" line. Now it just prints "... not found.